src/main/webapp/data/userMapData.json

```diff
@@ -554,15 +569,20 @@
     ],
     "Japan": [
       "AdRoll",
+      "International University of Japan (IUJ)",
       "Showa Women's University"
     ],
     "Kazakhstan": [
-      "Method Coding School"
+      "Method Coding School",
```

Comment:
this is also unnecessary

Reply:
Hi @shaharyar-shamshi  , As you can see comma was not there so i added it.